src/pages/books.tsx

```diff
@@ -0,0 +1,381 @@
+                        gap: '64px',
+                        maxWidth: '1024px',
+                        mx: 'inc40',
+                    }}
+                >
+                    {featuredBooks.map(book => (
+                        <FeaturedBookCard key={book.title} {...book} />
+                    ))}
+                </div>
+                <div
+                    sx={{
+                        width: '100%',
+                        bg: 'black70',
+                        display: 'flex',
+                        justifyContent: 'center',
+                    }}
+                >
+                    <div
+                        sx={{
+                            my: 'inc60',
+                            display: 'flex',
+                            flexDirection: 'column',
+                            maxWidth: '1440px',
+                            mx: 'inc40',
+                            width: '100%',
+                            alignItems: 'start',
+                        }}
+                    >
+                        <TypographyScale
+                            variant="heading6"
+                            sx={{
+                                color: '#fff',
+                            }}
+                        >
+                            Related Books
+                        </TypographyScale>
+                        <Grid
+                            columns={[1, null, 2, 4]}
+                            gap={[48, null, null, 36, 64]}
+                            sx={{ width: '100%', marginTop: '24px' }}
+                        >
+                            {relatedBooks.map(book => (
+                                <RelatedBookCard key={book.title} {...book} />
+                            ))}
+                        </Grid>
+                    </div>
+                </div>
+            </div>
+        </div>
+    );
+};
+
+export default BooksPage;
+
+export const getStaticProps: GetStaticProps<{
+    featuredBooks: FeaturedBook[];
+    relatedBooks: Book[];
+}> = async () => {
+    const featuredBooks: FeaturedBook[] = [
+        {
+            title: 'Practical MongoDB Aggregations',
+            authors: ['Paul Done'],
+            publishedDate: 'Sept. 2023',
+            description: `This technical guide takes you on a data-driven journey by teaching you how to streamline data manipulation, resolve data processing bottlenecks, and optimize
+        pipelines. This book is your go-to resource for becoming proficient with the MongoDB
+        aggregation framework.`,
+            amazonLink:
+                'https://www.amazon.com/Practical-MongoDB-Aggregations-developing-aggregation/dp/1835080642/ref=tmm_pap_swatch_0?_encoding=UTF8&amp&qid=1694163751&amp&sr=8-3',
+            packtLink:
+                'https://www.amazon.com/Practical-MongoDB-Aggregations-developing-aggregation/dp/1835080642/ref=tmm_pap_swatch_0?_encoding=UTF8&amp&qid=1694163751&amp&sr=8-3',
```

Comment:
Will this be replaced with a packtlink?

Reply:
Yes, I don't have some of the final copy yet, so think will change.